src/docs/apidoc.js

```diff
@@ -9,7 +9,7 @@ const options = {
   },
   servers: [
     {
-      url: process.env.SWAGGER_URL // Port Number on this URL must match Server Port Number
+      url: process.env.SWAGGER_URL, // Port Number on this URL must match Server Port Number
     },
     {
       url: 'http://localhost:3000/',
```

Comment:
@Edcreation , what if here, one changes the App PORT to another number, not 3000? I suggest here you use  this: 
```
url: `http://localhost:${process.env.PORT}/`
 ``` 
:  just to be flexible with whatever port used!

Reply:
Working on it 🚀

Reply:
> @Edcreation , what if here, one changes the App PORT to another number, not 3000? I suggest here you use this:
> 
> ```
> url: `http://localhost:${process.env.PORT}/`
> ```
> 
> : just to be flexible with whatever port used!

Done!🚀